validation-test/Runtime/Inputs/class-layout-from-objc/Classes.swift

```diff
@@ -0,0 +1,40 @@
+import Resilient
+import Foundation
+
+public class StaticClass: NSObject {
+  @objc var first: Int = 0
+  var forceUnalignment: Int8 = 0
+  var middle = GrowsToInt64()
+  @objc var last: Int = 0
```

Comment:
Why are these (non-public) ivars @objc?

Reply:
These are the ones I'm testing the ivar offsets of. It doesn't really matter if they're public or not; Swift never exposes direct compile-time ivar access to Objective-C.

Reply:
If we ever *don't* emit them for non-`@objc` ivars, I don't want the test to change. I think it's appropriate for something accessed through the ObjC runtime. (Besides, the non-`@objc` ivars aren't representable in Objective-C, and it feels weird to rely on them being there.)